README.md

```diff
@@ -34,4 +34,52 @@ This application is reusable and will convert any number of items into valid htm
 * Save the changes.
 * Exit out of the server from the terminal or cmd.
 * Run `node app.js` once again.
-* Refresh the browser and you will see your changes.
\ No newline at end of file
+* Refresh the browser and you will see your changes.
+
+## Style Guide
+This application adhears to the DWA Style Guide, which is built upon the AirBnB JavaScript style guide. To use the style guide make sure you have the following technologies.
+* [NPM](https://docs.npmjs.com/getting-started/installing-node)
+* [Mocha](https://mochajs.org/#installation)
+  * `npm install --global mocha`
+* [Chai](http://www.chaijs.com/guide/installation/#nodejs)
+  * `npm install chai`
+* [ESLint](https://eslint.org/docs/user-guide/getting-started) - Installed locally
+  * Install ESLint
+  `npm install esling --save-dev`
```

Comment:
I think this might be a typo.

`npm install eslint --save-dev` is what I believe it was supposed to say.

Reply:
Good catch! I'll fix that right away.